Edit chat Profile (#1687)
acterglobal/a3: .changes/1687-edit-chat-profile.md

> - [new] : Now user can update chat profile details (Name, Topic & Avatar) if they have required access.

Reviewer: ```suggestion
- [new]: You can now update chat profile details (Name, Topic & Avatar) for chats you are admin of
```
